stubs/pkg_resources/pkg_resources/__init__.pyi

```diff
@@ -254,7 +256,7 @@ empty_provider: EmptyProvider
 class FileMetadata(EmptyProvider, IResourceProvider):
     def __init__(self, path_to_pkg_info: str) -> None: ...
 
-def parse_version(v: str) -> Tuple[str, ...]: ...
+def parse_version(v: str) -> Union[Version, LegacyVersion]: ...
```

Comment:
Here it can be just `Version`, since `Union` is discouraged and `LegacyVersion` is deprecated